Cinema system/Cinema system/Models/Seat.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace CinemaSystem.Models
+{
+    public partial class Seat
+    {
+        public Seat()
+        {
+            Order = new HashSet<Order>();
+        }
+
+        public int Id { get; set; }
+        public string TypeOfSeat { get; set; }
+        public int Row { get; set; }
+        public int Place { get; set; }
+        public int HallId { get; set; }
+        public bool IsDeleted { get; set; }
+
+        public virtual Hall Hall { get; set; }
+        public virtual ICollection<Order> Order { get; set; }
```

Comment:
What is the purpose of this field? You can use one way navigation properties, e.g. Order having collection of Seats makes sense, but Seat having collection of Orders most probably not.